From 8302d6da81f950d498d8a7c70aa0e56376055057 Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Fri, 1 Sep 2017 10:53:09 +0200
Subject: [PATCH 83/93] SPEC: Fix unowned directory
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

https://fedoraproject.org/wiki/Packaging:UnownedDirectories

sh$ rpm -qf /usr/lib64/sssd/conf/ /usr/lib64/sssd/conf/sssd.conf
file /usr/lib64/sssd/conf is not owned by any package
sssd-common-1.15.3-2.fc27.x86_64

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
---
 contrib/sssd.spec.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contrib/sssd.spec.in b/contrib/sssd.spec.in
index cdab9874b43b87ac897e80ef7e81630a132a15d9..43b853bb523bb212316f3d0046da6b88e4505ad2 100644
--- a/contrib/sssd.spec.in
+++ b/contrib/sssd.spec.in
@@ -981,6 +981,7 @@ done
 %config(noreplace) %{_sysconfdir}/rwtab.d/sssd
 %dir %{_datadir}/sssd
 %{_sysconfdir}/pam.d/sssd-shadowutils
+%dir %{_libdir}/%{name}/conf
 %{_libdir}/%{name}/conf/sssd.conf
 
 %{_datadir}/sssd/cfg_rules.ini
-- 
2.14.1

